dist/aframe-altspace-component.js

```diff
@@ -597,9 +603,50 @@
 	    * @example <a-box n-mesh-collider></a-box>
 	    */
 		AFRAME.registerComponent('n-mesh-collider', {
-			init:nativeComponentInit,
-			remove: nativeComponentRemove,
-			update: nativeComponentUpdate,
+			_forEachMesh: function (func) {
+				var obj = this.el.object3DMap.mesh;
+				if (!obj) { return; }
+				if (obj instanceof THREE.Mesh) {
+					func(obj);
+				} else {
+					obj.traverse(function (childObj) {
+						if (childObj instanceof THREE.Mesh) {
+							func(childObj);
+						}
+					}.bind(this));
+				}
+			},
+			_initObj: function () {
+				this._forEachMesh(function (mesh) {
+					meshInit.call(this, mesh);
```

Comment:
this might lead to some unexpected collision meshes, as each one is getting its own convex hull (imagine a point cloud in one method or the other), should probably just note in the docs. I could also see this changing behavior in the future (as it's likely going to be inefficent)